web/src/lib/utils/helpers.ts

```diff
@@ -239,6 +240,17 @@ export const getConfigFieldByLegalStructure = (
   }
 };
 
+export const isFormedOutsideNavigator = (business: Business | undefined): boolean => {
```

Comment:
[pebble] I think this would be better to live in its own file in `domain-logic` folder. And have its own little unit test.

Reply:
Does it kinda felt redundant given that I'm basically testing all the same logic one level higher when I check if the Note Alert exists?

I'm already doing the tests and plan on finishing but I'm curious if you feel like there is a point where it becomes a poor idea